### PR TITLE
Enable OpenDDS_QOS_XML_XSC_Handler when xerces3 is available, regardless of dds4ccm

### PR DESCRIPTION
**Pull request by mitza-oci**
Contributed by Johnny Willemsen

### CODE REVIEW

File: dds/DCPS/QOS_XML_Handler/optional_ace_xml_utils.mpb [hunk 1]
```diff
@@ -1,2 +1,2 @@
-feature(dds4ccm_opendds): ace_xml_utils {
+feature(dcps_xml_handler): ace_xml_utils {
```
Comment (mitza-oci) [inline dds/DCPS/QOS_XML_Handler/optional_ace_xml_utils.mpb]: @jwillemsen where is dcps_xml_handler used as an MPC feature?  Should this mpb file just be removed?

**Comment by jwillemsen** [comment]
Yes, I will remove that (and some more base projects that where for 2.0a support) with another pull request.

**Comment by mitza-oci** [comment]
What is the purpose of doing another pull request to master?  Let's have one merge to master with all that's needed to make this a coherent change.  You can make a pull request targeted at the qos_xml branch to change this.

**Comment by jwillemsen** [comment]
I am working currently at the AXCIOMA level and have now a stable tree. Didn't want to start picking changes from other repositories, by doing it in two sets it is way easier to handle in terms of merging and spending time on this.

**Comment by jwillemsen** [comment]
Trying to merge this all together

**Comment by jwillemsen** [comment]
Merged all together to #427

**Comment by jwillemsen** [comment]
Cleanup looks ok, probably too much taken from dance in the past

**Comment by jwillemsen** [comment]
Merged your simplifications also in my #427

**Pull request merged**